apps/ffdweb-site/src/app/_styles/globals.css

```diff
@@ -209,7 +209,7 @@
 
   /* COPY TO CLIPBOARD */
   .copy-to-clipboard-button {
-    @apply hover:text-brand-primary-400;
+    @apply hover:text-brand-primary-400 p-2;
```

Comment:
Had to remove the `TOUCH_TARGET.touchAreaPadding` from the `CopyToClipboard` component as it is different for filecoin-site. Alternatively we can pass a prop to modify the TouchTarget padding?

Reply:
Can't we keep it the same? It seems like a very small change?